### DISCUSSION TITLE
Styling custom blocks

### DISCUSSION THREAD

**Discussion by web1100manager** [tailpress/tailpress#120] [Q&A]
Hi!

What are your thoughts on incorporating tailwind/tailpress in with building custom blocks for your theme? Each block would need it's own enqueued stylesheet, correct? So then the esbuild configuration would need to add all the additional stylesheets as entry points. Or am I thinking about this in the wrong way? Would love to hear some other opinions or thoughts!

Thank you

**Comment by jeffreyvr** [comment]
When I develop custom blocks I tend to use Laravel Mix instead of esbuild. 

You can set Mix as the compiler with the TailPress installer: `tailpress new example-theme --compiler="mix"`

For custom blocks, here is a basic setup:

https://gist.github.com/jeffreyvr/c1c81656fc826e15e0c223434e55065a

I don't necessarily create separate stylesheets for blocks.
